lib/ddtrace/contrib/active_model_serializers/events/serialize.rb

```diff
@@ -29,6 +29,15 @@ def event_name
           def span_name
```

Comment:
⬆️  I'm curious, why does this event name get a `!` prefix? I haven't seen any other named like this

Reply:
The original event name had a `!`: https://github.com/rails-api/active_model_serializers/blob/ff1a36ffbc82c6070905707b29978a7b897dea81/lib/active_model/serializable.rb#L5 :)

Reply:
Ah. Right you are :)